src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -132,115 +224,871 @@ static DamageTable D_8092C140 = {
+    this->actor.focus.rot.y = this->actor.shape.rot.y;
+    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->colliderCylinder.base);
+    if (this->invincibilityFrames == 0) {
+        CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->colliderCylinder.base);
+    } else {
+        this->invincibilityFrames--;
+    }
+    if (this->colliderQuad.base.atFlags & AT_ON) {
+        CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->colliderQuad.base);
+    }
+    if (this->actionFunc == EnIk_Block) {
+        f32 cos;
+        f32 sin;
+        Vec3f sp5C;
+        Vec3f sp50;
+        Vec3f sp44;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A020.s")
+        cos = Math_CosS(this->actor.shape.rot.y);
+        sin = Math_SinS(this->actor.shape.rot.y);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A124.s")
+        sp44.x = (this->actor.world.pos.x - (30.0f * cos)) + (20.0f * sin);
+        sp44.y = this->actor.world.pos.y;
+        sp44.z = this->actor.world.pos.z + (30.0f * sin) + (20.0f * cos);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A188.s")
+        sp50.x = this->actor.world.pos.x + (30.0f * cos) + (20.0f * sin);
+        sp50.y = this->actor.world.pos.y + 80.0f;
+        sp50.z = (this->actor.world.pos.z - (30.0f * sin)) + (20.0f * cos);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A28C.s")
+        sp5C.x = sp44.x;
+        sp5C.y = sp50.y;
+        sp5C.z = sp44.z;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A33C.s")
+        Collider_SetTrisVertices(&this->colliderTris, 0, &sp44, &sp50, &sp5C);
+        sp5C.x = sp50.x;
+        sp5C.y = sp44.y;
+        sp5C.z = sp50.z;
+        Collider_SetTrisVertices(&this->colliderTris, 1, &sp44, &sp5C, &sp50);
+        CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->colliderTris.base);
+    }
+    if (this->effectAlpha > 0.0f) {
+        if (this->drawEffect != 10) {
+            Math_StepToF(&this->effectAlpha, 0.0f, 0.05f);
+            this->effectScale = (this->effectAlpha + 1.0f) * 0.325f;
+            if ((this->effectAlpha + 1.0f) * 0.325f > 0.65f) {
+                this->effectScale = 0.65f;
+            } else {
+                this->effectScale = this->effectScale;
+            }
+        } else if (Math_StepToF(&this->steamScale, 0.65f, 0.01625f) == 0) {
+            func_800B9010(&this->actor, NA_SE_EV_ICE_FREEZE - SFX_FLAG);
+        }
+    }
+    EnIk_ArmorDestroyed(this, globalCtx);
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A570.s")
+s32 EnIk_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    EnIk* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A5BC.s")
+    if (this->drawArmorFlags != 0) {
+        if (D_8092C1A8[limbIndex] > 0) {
```

Comment:
Oh I've seen this before but can't remember what it's for... I think it's a mapping from limb to the body part positions or something?